Source/Urho3D/Graphics/Skeleton.h

```diff
@@ -110,14 +110,20 @@ class URHO3D_API Skeleton
 
     /// Return root bone.
     Bone* GetRootBone();
+    /// Return index of the bone by name hash. Return number of bones if not found.
+    unsigned GetBoneIndex(const StringHash& boneNameHash) const;

```

Comment:
Missing LuaScript and AngelScript bindings.

Reply:
Yeah, probably _some_ of these functions could be binded...